### ISSUE TITLE
table scroll is not working in foundation with flex

### ISSUE THREAD

**Issue by GingerBear** [zurb/foundation-sites#9658]
#### How to reproduce this bug:
- table scroll not working in foundation with flex http://codepen.io/gingerbear/pen/egdaLO
- table scroll works in foundation without flex http://codepen.io/gingerbear/pen/QdKRzY

#### What should happen:
- expecting table scroll works on foundation flex

#### What happened instead:
- table scroll doesn't work on foundation flex

#### version
- 6.3.0

**Comment by AidamUnlimited** [comment]
Hey, I had the same problem. After couple of hours of googling I tried to wrap it in an extra .row > .small-24.column and it worked!

So final markup here:
```
<div class="row expanded items-table">
	<div class="column"><!-- This is a part of my site layout. But instead of putting .table-scroll directly inside this column, I do the following: -->

		<div class="row">
			<div class="small-24 columns">
				<div class="table-scroll">
					<table>
						/table content
					</table>
				</div>
			</div>
		</div>

	</div>
</div>
```

**Comment by AidamUnlimited** [comment]
Experiment with min-width: 0; to parent .column, or even parent > parent .column

**Comment by kball** [comment]
Interesting... adding max-width: 100% to `.column` in flex mode appears to resolve... @brettsmason any reason you can think of why that would be a bad idea?

**Comment by kball** [comment]
tap @brettsmason and @rafibomb - any reason not to add max-width:100% to `.column` in flex mode? Even with the new grid in 6.4, would be great to get this fixed for 6.3.2

**Comment by IamManchanda** [comment]
Why not `min-width: 0` instead of `initial`??
http://codepen.io/IamManchanda/pen/qmgeQo

**Comment by brettsmason** [comment]
@IamManchanda That's already the case in develop: https://github.com/zurb/foundation-sites/commit/dc25d9666fad02478274dc3c9f1421caba9bc01f

@kball Yep that looks like a good fix to me.

**Comment by IamManchanda** [comment]
@brettsmason  ... Applogies .... didn't test out locallly ... just codepen! 
Yup i think  #9846 fixes this .... and should be in `6.3.2` !!

**Comment by brettsmason** [comment]
Yep will be fixed in 6.3.2/6.4 so closing this. Any problems though please re-open!
